pkg/cgroups/cgroups_unsupported.go

```diff
@@ -21,3 +24,8 @@ func UserOwnsCurrentSystemdCgroup() (bool, error) {
 func rmDirRecursively(path string) error {
 	return os.RemoveAll(path)
 }
+
+// UserConnection returns an user connection to D-BUS
+func UserConnection(uid int) (*systemdDbus.Conn, error) {
+	return nil, fmt.Errorf("systemd d-bus is not supported on this platform")
```

Comment:
`Errorf()` could probably just be `Error()`?

Reply:
Shouldn't the linter have blocked this?